packages/cli/lib/utils.js

```diff
@@ -266,3 +266,32 @@ exports.renameEJS = function() {
 
   return renameStream;
 };
+
+/**
+ * Get a validate function for object/array type
+ * @param {String} type 'object' OR 'array'
+ */
+exports.validateStringObject = function(type) {
+  return function validate(val) {
+    if (val === null || val === '') {
+      return true;
+    }
+
+    const err = `The value must be a stringified ${type}`;
```

Comment:
is this error message applicable to all types of errors we get below?

Reply:
Yep. This function comes from LB3 DataSource CLI command and it's the same error. 

Here's the original reference code: https://github.com/strongloop/generator-loopback/blob/master/lib/helpers.js#L210-L228